libs/mark-flow-ui/src/components/review.tsx

```diff
@@ -236,7 +236,13 @@ export function Review({
             footer={
               selectionsAreEditable && (
                 <Button tabIndex={-1} onPress={() => onChangeClick(contest.id)}>
-                  <Caption>
+                  <Caption
+                    style={{
+                      display: 'inline-flex',
+                      alignItems: 'center',
+                      gap: '0.25rem',
+                    }}
+                  >
```

Comment:
Did you do screenshots for the voter flow changes? I'm not seeing them so can't weigh in

Reply:
I think the only modified voter screens were this slight button change on the review screen and using "Continue" rather than "Okay" on the modal, so not going to prioritize getting screenshots right now.